pages/common/git-cherry.md

```diff
@@ -0,0 +1,17 @@
+# git cherry
+
+> Find commits yet to be applied to upstream.
+> Commits prefixed with - have an equivalent; those with + do not.
+> More information: <https://git-scm.com/docs/git-cherry>.
+
+- Show commits and their message with an equivalent in upstream:
```

Comment:
Perhaps this will read better?
```suggestion
- Show commits and their messages with the equivalent commit upstream:
```

Reply:
@sbrl `with the equivalent commit upstream` sounds to me like it's suggesting it shows a list of "commit + message" and "upstream commit" pairs, which it does not.  How about 

```suggestion
- Show commits (and their messages) with equivalent commits upstream:
```

Reply:
Ah, I see - I must has misunderstood. That looks better - let's go with that.